doc/cheatsheet.md

```diff
@@ -116,17 +164,32 @@ Note that whitespace is required after every `p'code'` operator; otherwise ni wi
     * Useful for accessing fields beyond the first 12, for example `$ ni <data> F_ 6..15`
     * `FM` is the number of fields in the row.
     * `FR n` is equivalent to `F_ n..FM`
-* Combining Perl with `r`
-  * `$ ni <data> rp'<...>'` - take rows where the Perl snippet `<...>` is truthy in Perl. 
-    * Be careful using `rp'...'` with numeric values, because `0` is not truthy in Perl; `$ ni n10 p'r a, 0' rpb` returns an empty stream.  
+
+##SSH and Containers
+
+* `C<container_name>[...]`: execute `[...]` in `<container_name>`
+  * Running in containers requires that Docker be installed on your machine.
+  * Running containers can be especially useful to take advantage of better OS-dependent utilities.
+  * For example, Mac OS X's `sort` is painfully slow compared to Ubuntu's. If you are developing on a Mac, there will be a noticeable performance difference increase by replacing:
+    * `ni n1E7 g` with
+    * `ni n1E7 Cubuntu[g]`
+  * Containers are also useful for testing the portability of your code.
+* `s<host>[...]`: execute `[...]` in `<host>`
+  * You will need to set up your hosts properly in your `.ssh/config` to use a named host. 
+  * You will want to do this to reduce keystrokes.
+  * Remember that within the bracketed operator, you will have access to the `<host>` filesystem.
+  * `ssh` zips its output before transfer over the network which will decrease overhead and increase speed in general (but not always).
```

Comment:
I think this depends on the specifics of the host connection, though it may be default by now (not sure).